examples/money.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1,user-scalable=no"/>
+    <meta name="format-detection" content="telephone=no"/>
+    <title>Your own kind of snowflakes</title>
+    <style>
+        html, body {
+            background-color: black;
+        }
+
+        .snowflake__inner:before {
+            background-image: url(https://cdn.shopify.com/s/files/1/1061/1924/products/Money_with_Wings_Emoji_large.png);
```

Comment:
The picture is very large (480x480). For example, it is better to use a free (open source) image. Without the use of external website (https://cdn.shopify.com).